src/fuzzy-picker.js

```diff
@@ -73,11 +73,13 @@ export default class FuzzyPicker extends React.Component {
       case 'Enter': { // Enter key
         let item = this.state.items[this.state.selectedIndex];
         if (item) {
+          this.state.items = this.getInitialItems();
```

Comment:
This should probably be `this.setState({items: this.getInitialItems()});` or something to that effect.

It could potentially be duplicative to have both this and the same statement below, but I don't think it's a big deal considering its one line. It could be worth refactoring into a function at some point though.